config/examples/BQ/Hephestos_2/Configuration.h

```diff
@@ -1001,6 +999,7 @@
 #define X_MAX_POS X_BED_SIZE
 #define Y_MAX_POS Y_BED_SIZE
 #define Z_MAX_POS 210
+#define Z_LOW_POINT -2 // Sets limit for moves below zero such as hotend tool offsets of probing
```

Comment:
The `Z_MIN_POS` setting exists as the lowest limit already. If users want to move below 0, they must set `Z_MIN_POS` below 0, but leave `Z_HOME_POS` set to 0 for their min endstop.

Reply:
I didnt notice the manual items down below. I do think a more intelligent method of setting the minimum travel point for probing and tool offsets is needed. Maybe using HOTEND_OFFSET_MAX as a +/- point and then using MIN (0-HOTEND_OFFSET_MAX, Z_MIN) for the softendstop.

Reply:
The `Z_PROBE_LOW_POINT` setting is used to abort probing only when the Z position is known, and the probe would go too far down. When the Z position is not known, we can't do anything at all to prevent probe damage in the case of a probe malfunction or mis-configuration.

Reply:
The intent was to prevent the hotend z offset application from being filtered out by the soft endstop clamping by allowing it the same motion below the home point as the probe move was allowed when position was known. Nothing intended for when position is not known. The other value was because the Z_MIN was used both to set home point and to set the soft endstop.

Reply:
Again, `Z_PROBE_LOW_POINT` is very specific, and refers not to a position, but a distance below the expected probe target. The extra idiot-proofing you want to do will need a new setting.